client/app/modals/tournament-status/tournament-status.component.ts

```diff
@@ -0,0 +1,72 @@
+import { Component, OnInit } from '@angular/core';
+import { TournamentStatusService } from '../../services/tournament-status.service';
+import { forkJoin } from '../../../../node_modules/rxjs';
+
+@Component({
+  selector: 'tournament-status',
+  templateUrl: './tournament-status.component.html',
+  styleUrls: ['./tournament-status.component.css']
+})
+export class TournamentStatusComponent implements OnInit {
+  private _matchChangeTime
```

Comment:
Why you have _ before each variable name?

Reply:
It is a convention to prefix private variables with _, it was what I did in the scoring module as well.

Reply:
It is a convention, but only when you don't have the private definition (i.e in JS), So its not needed here.